infrastructure/src/main/java/org/corfudb/infrastructure/ManagementServer.java

```diff
@@ -72,7 +72,8 @@
      * In milliseconds.
      */
     @Getter
-    private final long policyExecuteInterval = 1000;
+    @Setter
+    private volatile long policyExecuteInterval = 1000;
```

Comment:
No we would not want to set it at runtime.
I was setting it to reduce test run times. But I guess a better way to do this would be to pass it via server context at the time of management server construction.
Since this setter is not required right now, I am reverting it to how it was.